### PR TITLE
boost: Correct --disable-iconv option

### PR DESCRIPTION
**Pull request by datalogics-kam**
This was in the file as --disable-iconvv. I believe that was a typo.

Specify library name and version:  **boost/all**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/boost/config.yml [hunk 1]
```diff
@@ -1,4 +1,10 @@
 versions:
+    1.66.0:
+      folder: all
+    1.67.0:
+      folder: all
+    1.68.0:
+      folder: all
```
Comment (SSE4) [inline recipes/boost/config.yml:7]:
```suggestion
```

File: recipes/boost/all/conandata.yml [hunk 2]
```diff
@@ -53,6 +53,9 @@ sources:
     ]
     sha256: "83bfc1507731a0906e387fc28b7ef5417d591429e51e788417fe9ff025e116b1"
 patches:
+  1.66.0: []
+  1.67.0: []
+  1.68.0: []
```
Comment (SSE4) [inline recipes/boost/all/conandata.yml:58]:
```suggestion
```

**Comment by SSE4** [comment]
@danimtb @jgsogo seems like bot doesn't detect changes?

**Comment by danimtb** [comment]
@SSE4 indeed, there are no new changes in this PR. Check the diff here with @datalogics-kam's branch updated with the latest changes from master https://github.com/conan-io/conan-center-index/compare/master...danimtb:package/boost?expand=1

This PR is old and the fix was already contributed, so it can be closed!

**Pull request closed without merging**